.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_API_URL=
+SKIP_PREFLIGHT_CHECK=true
```

Comment:
add empty line at end of every file, to do this automatically you can get this done with prettier and have it auto fix on save

Reply:
added empty line at end of every file